changelog.yaml

```diff
@@ -16,7 +16,38 @@
 
 - version: "NEXT"
   date: TBD
-  
+  changes:
+  - type: enhancement
+    impact: minor
+    title: Make Jenkinsfile Runner properties configurable in PipelineRun custom resource objects.
+
+    description: >-

```

Comment:
We need to change all `>-` to `|-`. Don't touch the existing ones to avoid conflicts.